src/Page/PageServiceManager.php

```diff
@@ -126,7 +126,8 @@ protected function createResponse(PageInterface $page)
     {
         if ($page->getTarget()) {
             $page->addHeader('Location', $this->router->generate($page->getTarget()));
-            $response = new Response('', 302, $page->getHeaders() ?: []);
+            $redirectType = $page->getTargetRedirectType() ? $page->getTargetRedirectType() : 302;
+            $response = new Response('', $redirectType, $page->getHeaders() ?: []);
         } else {
```

Comment:
you can use `return` instead of `else` here.

Reply:
I'll change that. It was like this in the original file so I just made minimal changes.

Reply:
can you also fix flintCI errors

Reply:
Sure